packages/platform-sdk-ark/src/networks/compendia/mainnet.ts

```diff
@@ -1,36 +1,65 @@
 import { Coins } from "@arkecosystem/platform-sdk";
 
 const network: Coins.NetworkManifest = {
-	id: "compendia.mainnet",
+	uuid: "compendia.mainnet",
 	type: "live",
 	name: "Mainnet",
 	coin: "Compendia",
-	explorer: "https://bindscan.io/",
 	currency: {
 		ticker: "BIND",
 		symbol: "ß",
 	},
-	fees: {
-		type: "dynamic",
-		ticker: "BIND",
-	},
-	crypto: {
+	constants: {
 		slip44: 543,
-		signingMethods: {
-			mnemonic: true,
-			wif: true,
+	},
+	hosts: [
+		{
+			type: "full",
+			host: "https://apis.compendia.org",
 		},
-		expirationType: "height",
+		{
+			type: "explorer",
+			host: "https://bindscan.io/",
```

Comment:
Not related to this change, but a good opportunity to review: all urls that end with `/` get added an extra `/` when we build the final tx or block url in DW. So at least for explorer links we need to remove trailing slashes here.

Reply:
Yeah that's something I want to address by introducing a function that normalises the URLs. There's also a few that have query parameters which would need to be put at the end.